pkg/cmd/link/link.go

```diff
@@ -129,6 +133,19 @@ func (cmd *LinkCmd) run(info *LinkInfo) error {
 	}
 	info.PathWorkingDir = path
 
+	if len(info.remote) > 0 {
+		logger.Debug("flag remote", zap.Any("repository", info.remote))
+		urlFull, _ := regexp.MatchString(`^https?://(?:www\.)?(?:github\.com|bitbucket\.org|gitlab\.com)/[\w-]+/[\w-]+(\.git)?$`, info.remote)
+		fmt.Println("url", info.remote)
```

Comment:
Remove this

Reply:
sai dai kkkkkk 
![image](https://github.com/aziontech/azion/assets/52722027/3b843929-1c61-4a2c-866b-c9620351faa4)
acabei de ver validando

Reply:
Não teria q mostrar o repo que vai clonar ali na mensagem de link tmb?

Reply:
Acho q não ele já está informando o repositorio q é para ser clonado

Reply:
é que pela mensagem talvez deixa a entender que vai linkar outro diretório :p 
mas vou voltar para as férias agora

Reply:
Depois vemos isso, ai podemos pensar em outros steps q estão por vir mas vemos isso segunda, aproveite 📦

Reply:
Blza